packages/kolibri-tools/lib/webpack.config.base.js

```diff
@@ -71,7 +71,11 @@ module.exports = ({ mode = 'development', hot = false, cache = false, transpile
     rules.push({
       test: /\.(js|mjs)$/,
       loader: 'babel-loader',
-      exclude: { and: [/(node_modules\/vue|dist|core-js)/, { not: [/\.(esm\.js|mjs)$/] }] },
+      exclude: [
+        // \\ for Windows, / for macOS and Linux
```

Comment:
I don't fully understand how to interpret this comment in the context of the exclusion below.

Reply:
This is copy pasted from the babel docs - but it's just to make sure the path matching regex works on Windows and MacOS/Linux.